docs/Model_Assumptions.rst

```diff
@@ -63,6 +63,61 @@ In case of excessive excess energy, a warning is given that it seems to be cheap
 High excess energy can for example result into an optimized inverter capacity that is smaller than the peak generation of installed PV.
 This becomes unrealistic when the excess is very high.
 
+Stratified thermal storage
+##########################
+
+In order to model a stratified thermal energy storage the two parameters `fixed_losses_relative` and `fixed_losses_absolute` are passed through `storage_xx.csv`.
```

Comment:
```suggestion
In order to model a stratified thermal energy storage the two parameters `fixed_losses_relative` and `fixed_losses_absolute` are passed to `storage_xx.csv`.
```

You do not have to have linebreaks for each sentence, but you can.

Reply:
Please add something that explicitly states that the parameters are not required for `storage_*.csv`.